modules/widgets/wordpress-post-widget.php

```diff
@@ -734,26 +734,30 @@ public function widget( $args, $instance ) {
 		// Enqueue front end assets.
 		$this->enqueue_scripts();
 
-		echo $args['before_widget'];
+		$content = '';
+
+		$content .= $args['before_widget'];
 
 		if ( empty( $instance['url'] ) ) {
 			if ( current_user_can( 'manage_options' ) ) {
-				echo '<p>';
+				$content .= '<p>';
 				/* Translators: the "Blog URL" field mentioned is the input field labeled as such in the widget form. */
-				esc_html_e( 'The Blog URL is not properly setup in the widget.', 'jetpack' );
-				echo '</p>';
+				$content .= esc_html_e( 'The Blog URL is not properly setup in the widget.', 'jetpack' );
```

Comment:
The function `esc_html_e` outputs instead of returning and will break this.
Replace this with `esc_html__`.